src/closure-annotations.js

```diff
@@ -1,204 +0,0 @@
-/**
- * @suppress {duplicate}
- */
-var ensureFloat32;
-/**
- * @suppress {duplicate}
- */
-var ensureFloat64;
-/**
- * @suppress {duplicate}
- */
-var VoidPtr;
-
-/**
- * GL functions (https://github.com/emscripten-core/emscripten/commit/97a464a654fdadf5dfb8aa082b48516e6bf8d402#commitcomment-25520648)
- */
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDrawArrays;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDrawElements;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glActiveTexture;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glEnable;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDisable;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glTexEnvf;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glTexEnvi;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glTexEnvfv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glGetIntegerv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glIsEnabled;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glGetBooleanv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glGetString;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glCreateShader;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glShaderSource;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glCompileShader;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glAttachShader;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDetachShader;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glUseProgram;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDeleteProgram;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glBindAttribLocation;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glLinkProgram;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glBindBuffer;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glGetFloatv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glHint;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glEnableVertexAttribArray;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glDisableVertexAttribArray;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _emscripten_glVertexAttribPointer;
-
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _glTexEnvf;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _glTexEnvi;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _glTexEnvfv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _glGetTexEnviv;
-/**
- * @suppress {duplicate, undefinedVars}
- */
-var _glGetTexEnvfv;
-
-/**
- * @suppress {undefinedVars}
- */
-var emscripten_source_map;
```

Comment:
We either don't have a test that would prove the above annotation is needed, or the code has since added a `typeof emscripten_source_map === 'undefined')` check that is now found in library.js before assignment, making the above annotation redundant.

Reply:
Digging through git blame, `emscripten_source_map` was added in https://github.com/emscripten-core/emscripten/pull/5808/files , and that also added a test. So that shows that this annotation is no longer needed since the test passes.